sklearn/kernel_ridge/tests/test_krr.py

```diff
@@ -16,12 +14,6 @@
 Y = np.array([y, y]).T
 
 
-def test_kernel_ridge():
```

Comment:
Why remove this and other tests here? This seems to be the fundamental test of KernelRidge..

Reply:
Wow, sorry. Because you said most of these tests were tested in `sklearn/tests/test_common.py`, I tested with coverage which tests were useful or not. I run

```bash
pytest sklearn/kernel_ridge/ sklearn/tests/ --showlocals -v --cov=sklearn/kernel_ridge --cov-report=html:coverage
```

And check the coverage. Then, I removed tests that I thought they were superfluous, and run pytest again. If the number of covered lines hadn't decreased, I reached to the conclusion these tests were superfluous.

However, if you think those tests were essential, I'll add them

Reply:
But you are right, `test_kernel_ridge` was important, I added again 5a3bedb